bson/doc.go

```diff
@@ -70,8 +70,9 @@
 //     otherwise.
 //  4. int64 marshals to BSON int64.
```

Comment:
I recommend adding a reference to `Encoder.IntMinSize`, which can change this behavior.
```suggestion
//  4. int64 marshals to BSON int64 (unless [Encoder.IntMinSize] is set).
```